doc/src/manual/command-line-interface.md

```diff
@@ -69,7 +69,7 @@ Note that although you should have a `~/.julia` directory once you've run Julia
 first time, you may need to create the `~/.julia/config` folder and the
 `~/.julia/config/startup.jl` file if you use it.
 
-To have startup code run only in [The Julia REPL] (and not when `julia` is *e.g.* run
+To have startup code run only in [The Julia REPL](@ref) (and not when `julia` is *e.g.* run
```

Comment:
Can you use an `@id` at the anchor?

Reply:
Using `@id` at the anchor breaks other places in the manual that refer to it, expecting it to have the default name:

https://github.com/JuliaLang/julia/blob/b12ddca3b18341d2abca10b0858f554e36452cfb/doc/src/manual/workflow-tips.md?plain=1#L7

https://github.com/JuliaLang/julia/blob/b12ddca3b18341d2abca10b0858f554e36452cfb/doc/src/manual/environment-variables.md?plain=1#L344

I could change those two `@ref`s of course, but that seems like a change for a different PR?